docs/book/src/reference/metrics.md

```diff
@@ -69,6 +70,22 @@ NAME                                         AGE
 monitor-controller-manager-metrics-monitor   2m8s
 ```
 
+<aside class="warning">
+<h2>If you are using Prometheus Operator ensure that you have the required
+permissions</h2>
+
+If you are using Prometheus Operator then be aware that by default, its RBAC
+rules are only enabled for the `default` and `kube-system namespaces`. See its
+guide to know how to configure the
+[kube-prometheus](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/monitoring-other-namespaces.md)
+to monitor other namespaces with `.jsonnet`.
+
+However, you can also enable it using RBAC. See the Prometheus Operator
+[Getting Started](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#enable-rbac-rules-for-prometheus-pods)
```

Comment:
@machadovilaca for some reason when I press the link it doesn't get me to the |"Enable RBAC rules for Prometheus pods" section, eventough your link it to it. Can we change the name of the link from "Getting Stared" to "Enable RBAC rules for Prometheus pods" ?

Reply:
+1

Reply:
updated